ports/atmel-samd/boards/robohatmm1/pins.c

```diff
@@ -0,0 +1,82 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {	
+	// SERVO Pins (Primary)
+	{ MP_ROM_QSTR(MP_QSTR_SERVO1), MP_ROM_PTR(&pin_PA16) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO2), MP_ROM_PTR(&pin_PA17) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO3), MP_ROM_PTR(&pin_PA18) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO4), MP_ROM_PTR(&pin_PA19) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO5), MP_ROM_PTR(&pin_PA11) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO6), MP_ROM_PTR(&pin_PA10) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO7), MP_ROM_PTR(&pin_PA09) },
+	{ MP_ROM_QSTR(MP_QSTR_SERVO8), MP_ROM_PTR(&pin_PA08) },
+	
+	// RC_CH Pins (Primary)
+	{ MP_ROM_QSTR(MP_QSTR_RCH1), MP_ROM_PTR(&pin_PA07) },
+	{ MP_ROM_QSTR(MP_QSTR_RCH2), MP_ROM_PTR(&pin_PA06) },
+	{ MP_ROM_QSTR(MP_QSTR_RCH3), MP_ROM_PTR(&pin_PA05) },
+	{ MP_ROM_QSTR(MP_QSTR_RCH4), MP_ROM_PTR(&pin_PA04) },
+	
+	// Special Function (Primary)
+	{ MP_ROM_QSTR(MP_QSTR_I2C_SDA), MP_ROM_PTR(&pin_PA00) },
+	{ MP_ROM_QSTR(MP_QSTR_I2C_SCL), MP_ROM_PTR(&pin_PA01) },
+	
+	{ MP_ROM_QSTR(MP_QSTR_BATGUAGE), MP_ROM_PTR(&pin_PA02) },
```

Comment:
IIRC we've standardized on `BATTERY` for this name.

Reply:
No worries.  We can change that.

Reply:
Ok, cool. Let me know when I should review.

Reply:
Should be all good now.  Could you please point me in the right direction for the .travis.yml file you mentioned?  I'm new at this 😄 .

Reply:
>  Could you please point me in the right direction for the .travis.yml

First board build line is here: https://github.com/adafruit/circuitpython/blob/master/.travis.yml#L24

Use one of the "TRAVIS_BOARDS" environment variable to add your board to.

Reply:
Thank you!  All done.